api/HTMLVideoElement.json

```diff
@@ -52,7 +52,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/getVideoPlaybackQuality",
           "support": {
             "chrome": {
-              "version_added": "23"
+              "version_added": "80"
```

Comment:
Chrome Android is missing.

Reply:
Also, Firefox removes `corruptedVideoFrames` in version 73.

Reply:
> Chrome Android is missing.

oops — thanks for catching that; added in https://github.com/mdn/browser-compat-data/pull/5452/commits/cb48157

Reply:
> Also, Firefox removes `corruptedVideoFrames` in version 73.

Thanks for the heads-up. https://github.com/mdn/browser-compat-data/pull/5452/commits/79558e4 adds `version_removed` for that and also makes other fixes/updates to the data — to reflect that `corruptedVideoFrames`:

* was never actually implemented in Chrome (as far as I can tell — at least not in any way exposed to devs; at https://github.com/w3c/media-playback-quality/pull/20#issuecomment-572848608 I’ve asked for confirmation)
* has been marked as deprecated in the spec https://w3c.github.io/media-playback-quality/#dom-videoplaybackquality-corruptedvideoframes

Reply:
It shipped in 80. https://storage.googleapis.com/chromium-find-releases-static/6c2.html#6c2d80337cb1988e53ae29f7418b52e9d81dec8e

If it hadn't, I would have asked that the version number be removed.

Reply:
>> `corruptedVideoFrames`
>
> It shipped in 80. https://storage.googleapis.com/chromium-find-releases-static/6c2.html#6c2d80337cb1988e53ae29f7418b52e9d81dec8e

OK, thanks — pushed 0d19769 to update that